perses/annihilation/new_relative.py

```diff
@@ -0,0 +1,1351 @@
+            sterics_energy_expression, electrostatics_energy_expression = self._nonbonded_custom_cutoff(epsilon_solvent, r_cutoff)
+        elif self._nonbonded_method in [openmm.NonbondedForce.PME, openmm.NonbondedForce.Ewald]:
+            [alpha_ewald, nx, ny, nz] = self._old_system_forces['NonbondedForce'].getPMEParameters()
+            delta = self._old_system_forces['NonbondedForce'].getEwaldErrorTolerance()
+            r_cutoff = self._old_system_forces['NonbondedForce'].getCutoffDistance()
+            sterics_energy_expression, electrostatics_energy_expression = self._nonbonded_custom_ewald(alpha_ewald, delta, r_cutoff)
+        else:
+            raise Exception("Nonbonded method %s not supported yet." % str(self._nonbonded_method))
+        sterics_energy_expression += self._nonbonded_custom_sterics_common()
+        electrostatics_energy_expression += self._nonbonded_custom_electrostatics_common()
+
+        sterics_mixing_rules, electrostatics_mixing_rules = self._nonbonded_custom_mixing_rules()
+
+        # Create CustomNonbondedForce to handle interactions between alchemically-modified atoms and rest of system.
+        electrostatics_custom_nonbonded_force = openmm.CustomNonbondedForce("U_electrostatics;" + electrostatics_energy_expression + electrostatics_mixing_rules)
+        electrostatics_custom_nonbonded_force.addGlobalParameter("lambda_electrostatics", 0.0)
+        electrostatics_custom_nonbonded_force.addGlobalParameter("softcore_beta", self.softcore_beta)
+        electrostatics_custom_nonbonded_force.addPerParticleParameter("chargeA") # partial charge initial
+        electrostatics_custom_nonbonded_force.addPerParticleParameter("chargeB") # partial charge final
+
+        self._hybrid_system.addForce(electrostatics_custom_nonbonded_force)
+        self._hybrid_system_forces['core_electrostatics_force'] = electrostatics_custom_nonbonded_force
+
+        sterics_custom_nonbonded_force = openmm.CustomNonbondedForce("U_sterics;" + sterics_energy_expression + sterics_mixing_rules)
+        sterics_custom_nonbonded_force.addGlobalParameter("lambda_sterics", 0.0)
+        sterics_custom_nonbonded_force.addGlobalParameter("softcore_alpha", self.softcore_alpha)
+        sterics_custom_nonbonded_force.addPerParticleParameter("sigmaA") # Lennard-Jones sigma initial
+        sterics_custom_nonbonded_force.addPerParticleParameter("epsilonA") # Lennard-Jones epsilon initial
+        sterics_custom_nonbonded_force.addPerParticleParameter("sigmaB") # Lennard-Jones sigma final
+        sterics_custom_nonbonded_force.addPerParticleParameter("epsilonB") # Lennard-Jones epsilon final
+
+
+
+        self._hybrid_system.addForce(sterics_custom_nonbonded_force)
+        self._hybrid_system_forces['core_sterics_force'] = sterics_custom_nonbonded_force
+
+        #Add a regular nonbonded force for all interactions that are not changing.
+        standard_nonbonded_force = openmm.NonbondedForce()
```

Comment:
If this was missed, it means the tests are not catching things that are sensitive to this. Can you add a test that is not just a molecule in vacuum, but maybe is a small molecule in some environment that contains other interacting particles that aren't part of the alchemical region? It could even be a larger molecule where only some atoms are involved in the alchemical part.